src/libs/ReportUtils.ts

```diff
@@ -3065,10 +3066,41 @@ function getInvoicePayerName(report: OnyxEntry<Report>): string {
     return getPolicyName(report, false, allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${invoiceReceiver?.policyID}`]);
 }
 
+/**
+ * Parse html of reportAction into text
+ */
+function parseReportActionHtmlToText(reportAction: OnyxEntry<ReportAction>, reportID: string): string {
+    const {html, text} = reportAction?.message?.[0] ?? {};
+
+    if (!html) {
+        return text ?? '';
+    }
+
+    const mentionReportRegex = /<mention-report reportID="(\d+)" *\/>/gi;
+    const matches = html.matchAll(mentionReportRegex);
+
+    const reportIdToName: Record<string, string> = {};
+    for (const match of matches) {
+        if (match[1] !== reportID) {
+            // eslint-disable-next-line @typescript-eslint/no-use-before-define
+            reportIdToName[match[1]] = getReportName(getReport(match[1])) ?? '';
+        }
+    }
+
+    const mentionUserRegex = /<mention-user accountID="(\d+)" *\/>/gi;
+    const accountIdToName: Record<string, string> = {};
```

Comment:
```suggestion
    const accountIDToName: Record<string, string> = {};
```

Reply:
and also with `accountIdToName` in [here](https://github.com/Expensify/expensify-common/blob/c3cc2c0d07c961df08a7147b9c4405014521b8bd/lib/ExpensiMark.js#L552C53-L552C68)